lib/web_catalog/api_extractor.es6.js

```diff
@@ -247,10 +252,16 @@ foam.CLASS({
           this.RemoveInterfacesProcessor.create({
             otherURLs: [
               // For interface name "a":
-              'https://github.com/GoogleChromeLabsLabs/confluence/issues/299#issuecomment-391119693',
+              'https://github.com/GoogleChromeLabs/confluence/issues/299#issuecomment-391119693',
+              // For interface name "__REACT_DEVTOOLS_GLOBAL_HOOK__":
```

Comment:
I think this belongs on the newly added `RemoveApisProcessor` below; setting [this property](https://github.com/GoogleChromeLabs/confluence/blob/578b00eae6a0b04409b7360ddb2409fb30704ef6/lib/web_catalog/post_processors.es6.js#L37-L42) on the appropriate instance. (Should be, as here, an array with one element.)